java_fibonacci.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class java_fibonacci{
```

Comment:
Codacy found an issue: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6353698)